pkg/command/context.go

```diff
@@ -182,9 +182,15 @@ var createCtxCmd = &cobra.Command{
     and release of new services (and CLI plugins) which extend and combine features provided by
     individual tanzu components.
 
-    Note: To create Mission Control (TMC) or Tanzu contexts, an API Key is required. The API Key
-    value can be provided with the environment variable TANZU_API_TOKEN or can be entered as input
-    while creating the context.
+    Notes: 
+    1. TMC context: To create Mission Control (TMC) context an API Key is required. It can be provided using the 
+       TANZU_API_TOKEN environment variable or entered during context creation.
+    2. Tanzu context: To create Tanzu context an API Key is optional.If provided using the TANZU_API_TOKEN environment variable,
+       it will be used; otherwise, the Tanzu CLI will fall back to the interactive login flow. Note that the interactive login 
```

Comment:
```suggestion
       it will be used. Ttherwise, the CLI will attempt to log in interactively to the user's default Cloud Services organization.  You can override or choose a custom organization by setting the 
```

Reply:
Thanks. Updated the help.